.eslintrc.js

```diff
@@ -1,6 +1,6 @@
-{
+module.exports = {
   "extends": [
-    "react-app",
```

Comment:
eslint-config-react-app and eslint-plugin-import came into conflict after upgrade